examples/prism-elisp.html

```diff
@@ -0,0 +1,54 @@
+<h1>Elisp</h1>
+<p>To use this language, use one of the following classes:</p>
+<ul>
+	<li>"language-elisp"</li>
+	<li>"language-lisp"</li>
+	<li>"language-emacs"</li>
+</ul>
```

Comment:
Same thing here: let's use `Lisp` as the main language. And move the class `"language-lisp"` in first position.